fastlane/Fastfile

```diff
@@ -0,0 +1,123 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+fastlane_require 'fileutils'
+
+default_platform(:ios)
+
+workspace_name = "MailchimpSDK.xcworkspace"
+sdk_scheme_name = "MailchimpSDK"
+build_path = "../build"
+deploy_path = "../deploy"
+device_archive_path = "#{build_path}/MailchimpDeviceSDK.xcarchive"
+simulator_archive_path = "#{build_path}/MailchimpSimulatorSDK.xcarchive"
+framework_name = "MailchimpSDK"
+xcframework_file_name = "Mailchimp.xcframework"
+xcframework_output_path = "#{build_path}/#{xcframework_file_name}"
+deploy_zip = "Mailchimp-SDK-iOS.zip"
+changelog_path = "../Changelog"
+readme_path = "../README.md"
+license_path = "../LICENSE"
+
+platform :ios do
+  desc "Builds the SDK framework"
+  lane :build_framework do
+    xcodebuild()
+  end
+
+  desc "Archives the SDK framework"
+  lane :archive_framework do
+    # Defining 2 destinations in a single call did not work in practice. This will build the 2 archives separately.
+    sh("xcodebuild archive -workspace '../#{workspace_name}' -archivePath '#{device_archive_path}' -scheme #{sdk_scheme_name} -destination 'generic/platform=iOS' SKIP_INSTALL=NO BUILD_LIBRARY_FOR_DISTRIBUTION=YES")
+    sh("xcodebuild archive -workspace '../#{workspace_name}' -archivePath '#{simulator_archive_path}' -scheme #{sdk_scheme_name} -destination 'generic/platform=iOS Simulator' SKIP_INSTALL=NO BUILD_LIBRARY_FOR_DISTRIBUTION=YES")
+  end
+
+  desc "Create Swift Binary Framework"
+  lane :create_binary_framework do
+    archive_framework
+
+    # Remove any existing xcframeworks on local machine
+    sh("rm -rf #{xcframework_output_path}")
+
+    # When Xcode 11 is officially released, fastlane may support -create-xcframework in the xcodebuild() command which could replace this (https://github.com/fastlane/fastlane/issues/15055)
```

Comment:
Is this still true? I think fastlane has had that issue opened/closed multiple times.

Reply:
I can't find any indication from fastlane that this is supported yet.

Reply:
Same :(